components/feature/addons/src/main/res/values/strings.xml

```diff
@@ -153,6 +157,14 @@
     <string name="mozac_feature_addons_successfully_uninstalled">Successfully uninstalled %1$s</string>
     <!-- Text shown after failed to uninstall an add-on. %1$s is the add-on name. -->
     <string name="mozac_feature_addons_failed_to_uninstall">Failed to uninstall %1$s</string>
+    <!-- Text shown after failed to allow an add-on in private browsing mode. %1$s is the add-on name. -->
+    <string name="mozac_feature_addons_failed_to_allow_in_private_browsing">Failed to allow %1$s in private browsing mode</string>
```

Comment:
Hm I wonder if displaying these strings is really needed? We could confirm with Bram, but it seems too much. Wdyt. Wouldn't it be ok to just have the slider like desktop and not display a toast?

Reply:
Agreed.